.github/workflows/olm.yml

```diff
@@ -0,0 +1,38 @@
+name: OLM
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        go: [ '1.15' ]
+    name: Go ${{ matrix.go }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      id: go
+      with:
+        go-version: ${{ matrix.go }}
+
+    - name: Create and set up K8s KinD Cluster
+      run: |
+        ./scripts/kind-with-registry.sh
+        make deploy-olm
+
+    - name: Build and Push Test Container Image and Bundle/Bundle Index Images
+      run: make build-and-push-bundle-images IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA REPO=localhost:5000 VERSION=$GITHUB_SHA OPERATOR_VERSION=0.0.1
```

Comment:
Is there a reason to set `OPERATOR_VERSION=0.0.1` here? It will match the current version of the CSV, but this will need to be updated every time the version of the CSV is updated. Can we just use `VERSION` that is being set to the `GITHUB_SHA`?